x/rollapp/types/message_create_rollapp.go

```diff
@@ -9,6 +9,8 @@ const TypeMsgCreateRollapp = "create_rollapp"
 
 var _ sdk.Msg = &MsgCreateRollapp{}
 
+const MaxAllowedSequencers = 100
```

Comment:
create an issue please to make it a param

Reply:
done